zerver/views/realm.py

```diff
@@ -60,139 +73,118 @@ def parse_jitsi_server_url(
 JITSI_SERVER_URL_MAX_LENGTH = 200
 
 
+def check_jitsi_url(value: str) -> str:
+    var_name = "jitsi_server_url"
+    value = check_string(var_name, value)
+
+    if value in list(Realm.JITSI_SERVER_SPECIAL_VALUES_MAP.keys()):
+        return value
+
+    validator = check_capped_url(JITSI_SERVER_URL_MAX_LENGTH)
+    try:
+        return validator(var_name, value)
+    except ValidationError:
+        raise JsonableError(_("{var_name} is not an allowed_type").format(var_name=var_name))
```

Comment:
Done this to make sure error is mostly the same in the cases as it was before. Maybe we can update it to just be something like "It is not a url or is too long.".

Reply:
Yeah maybe worth a quick "api design" thread about this detail.

Reply:
Started https://chat.zulip.org/#narrow/stream/378-api-design/topic/error.20message.20while.20updating.20.60jitsi_server_url.60.20setting